link-all.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
please put under `tools` and add an npm script in the root `package.json` file to invoke. Also update the contribution guide.

Reply:
Maybe we shouldn't add it to package.json? This script is supposed to be called from other packages.